src/agent/sqlite_manager/include/sqlite_manager.hpp

```diff
@@ -3,15 +3,40 @@
 #include <SQLiteCpp/SQLiteCpp.h>
 #include <SQLiteCpp/VariadicBind.h>
 
+#include <mutex>
 #include <string>
+#include <vector>
 
 namespace sqlite_manager
 {
-    class Column
+    const std::map<int, std::string> mapColTypeString {
```

Comment:
This could be a private, hidden implementation detail. To clients of these class we should have our own definitions.

Maybe an `enum class` called `ColumnType`.